www/index.html

```diff
@@ -171,6 +204,16 @@
 
   <hr/>
 
+  <div>
+    <p>
+      If you are running Windows on Arm,<br/>download and run
+      <a class="windows-download" href="https://win.rustup.rs/aarch64">rustup&#x2011;init.exe</a>
```

Comment:
Before we merge this, I think we should let @rust-lang/infra have a look at the artifact deployment: it looks like the URL is invalid now, but I guess the URL itself looks perfectly fine.

Reply:
I think I’ll split off the removal of the Ring workaround into a separate PR then, so that it could be merged sooner. Then this PR will be exclusively about the delivery.

Reply:
@Alovchin91 Do you think it's okay to remove the `# skip-pr` in the following line:
 
https://github.com/rust-lang/rustup/blob/2a5a69e0914ff419554d684ca71eb1d72c72bcb3/ci/actions-templates/windows-builds-template.yaml#L23

... so that we might gain more confidence about the quality of the build? Currently no tests will be run for this artifact, but the fact that it compiles means something too at least...

Reply:
Yup, sounds good to me. Done!

Reply:
Also split the Ring part off into #3845